sklearn/linear_model/tests/test_base.py

```diff
@@ -321,6 +321,15 @@ def test_csr_preprocess_data():
     assert_equal(csr_.getformat(), 'csr')
 
 
+def test_csr_preprocess_data_no_checks():
```

Comment:
Maybe better to parametrize the `test_csr_preprocess_data` above with,
```py
@pytest.mark.parametrize('check_data', (True, False))
def test_csr_preprocess_data(check_data):
    ...
```
as it's the same test..

Reply:
The test added wasn't covering all the cases we should cover. So have updated the test instead. Don't think they should be merged, but please take another look and let me know.